lib/helpers/results_helpers.js

```diff
@@ -462,7 +463,7 @@ module.exports = class ResultsHelpers {
       prettyResult += `${currentIndentation}}`;
       return prettyResult;
     }
-    if (result) {
+    if (result || result === 0) {
```

Comment:
Results of aggregate functions being ignored if it is 0(https://oncprojectracking.healthit.gov/support/browse/BONNIEMAT-1096)